src/test/sysinfo/test_sysinfo.rs

```diff
@@ -0,0 +1,16 @@
+/*
+ * The Shadow Simulator
+ * See LICENSE for licensing information
+ */
+
+fn main() {
+    let mut info = unsafe { std::mem::zeroed::<libc::sysinfo>() };
+    let info_ptr = &mut info as *mut libc::sysinfo;
+    let rv = unsafe { libc::sysinfo(info_ptr) };
+
+    println!("Found return value {:?}.", rv);
+    assert_eq!(rv, 0);
+    println!("Found uptime {:?}.", rv);
```

Comment:
Should this be `info.uptime` instead of `rv`?

Reply:
Oops good catch!!